Paintroid/src/main/java/org/catrobat/paintroid/ui/button/ColorButton.java

```diff
@@ -40,7 +40,11 @@
 	private static final int RECT_BORDER_COLOR = Color.LTGRAY;
 	private static final boolean DEFAULT_DRAW_SELECTED_COLOR = true;
 
-	private Paint colorPaint;
+	public static Paint getColorPaint() {
+		return colorPaint;
+	}
+
+	private static Paint colorPaint;
```

Comment:
Please revert the changes to `colorPaint`, this doesn't have to be static.
`getColorPaint()` is also not needed as far as I can see.
It seems that there are some functions and variables in here that are not needed anymore:
* `resetDrawSelectedColor()`
* `getDrawSelectedColor()`
* `setDrawSelectedColor()`
* `DEFAULT_DRAW_SELECTED_COLOR`

can you remove those too?